### PR TITLE
Full-form subcommands and completion

### PR DESCRIPTION
**Pull request by Xophmeister**
* Move to using full-form subcommands (e.g., `format` rather than `fmt`), for clarity. (The short-forms are still valid aliases).
* Added a shell completion generation subcommand.

### CODE REVIEW

File: default.nix
Comment (ErinvanderVeen) [inline default.nix:1]: What do did you use to format this nix file?

Comment (Xophmeister) [reply default.nix:1]: My editor is set to use `nixpkgs-fmt` for Nix files. I don't remember why it's set that way, so I assume other formatters would also work if that's preferable :sweat_smile:

Comment (Xophmeister) [reply default.nix:1]: ...The formatters that my linter supports for Nix are: Alejandra, `nixfmt` and `nixpkgs-fmt`.

Comment (Niols) [reply default.nix:1]: I remember using `nixfmt` everywhere in the HASG, but I don't know if it's global at Tweag or if each project does their own thing.

Comment (Niols) [reply default.nix:1]: Actually, I think I remember a debate on this on Tweag's Slack being unconclusive.

Comment (ErinvanderVeen) [reply default.nix:1]: This is what happens when an ecosystem has multiple formatters!

**Review by ErinvanderVeen — APPROVED** [review]

**Pull request merged**